programs/protocol/src/actions/carrier/accept_offer.rs

```diff
@@ -14,13 +12,13 @@ pub struct AcceptOffer<'info> {
     pub task: AccountLoader<'info, AcceptedOffer>,
     #[account(mut,
         seeds = [b"offer", carrier.load().unwrap().creator.as_ref(), &offer.load().unwrap().no.to_le_bytes()], bump,
-        // close = signer,
+        close = signer,
```

Comment:
why not keep it for stats?

Reply:
I don't think there is any data deleted here

Reply:
but test say the other way

Reply:
in test you're asserting null after accepting the offer. can u explain?